Plots/Panels/panel_3.py

```diff
@@ -0,0 +1,198 @@
+    )
+
+    # Label the contours
+    ax.clabel(
+        handles["contour"], fontsize="small", fmt="%.0f",  # Turn off decimal points
+    )
+
+    return handles
+
+
+###############################################################################
+# Here's how this function works
+
+ax = plt.gca(projection=ccrs.PlateCarree())
+plot_labelled_filled_contours(ds.U, ax)
+
+###############################################################################
+# These next two functions add nice axes decorations and make the plot look more
+# like NCL
+
+def add_lat_lon_ticklabels(ax):
+    """
+    Nice latitude, longitude tick labels
+    """
+    from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+    lon_formatter = LongitudeFormatter(zero_direction_label=True)
+    lat_formatter = LatitudeFormatter()
+    ax.xaxis.set_major_formatter(lon_formatter)
+    ax.yaxis.set_major_formatter(lat_formatter)
+
+
+def nclize_axis(ax):
+    """
+    Utility function to make plots look like NCL plots
+    """
+    import matplotlib.ticker as tic
+
+    ax.tick_params(labelsize="small")
+    ax.minorticks_on()
+    ax.xaxis.set_minor_locator(tic.AutoMinorLocator(n=3))
+    ax.yaxis.set_minor_locator(tic.AutoMinorLocator(n=3))
+
+    # length and width are in points and may need to change depending on figure size etc.
+    ax.tick_params(
+        "both",
+        length=8,
+        width=1.5,
+        which="major",
+        bottom=True,
+        top=True,
+        left=True,
+        right=True,
+    )
+    ax.tick_params(
+        "both",
+        length=5,
+        width=0.75,
+        which="minor",
+        bottom=True,
+        top=True,
+        left=True,
+        right=True,
+    )
+
+
+###############################################################################
+# Now we'll make two panels (subplots in matplotlib terminology) using ``plt.subplots``
+# We'll specify ``constrained_layout=True`` which will attempt to automatically
+# layout panels, colorbars and axes decorations nicely.
+# https://matplotlib.org/tutorials/intermediate/constrainedlayout_guide.html
+
+f, ax = plt.subplots(
+    2,  # 2 rows
+    1,  # 1 column
+    constrained_layout=True,  # "magic"
+    subplot_kw={"projection": ccrs.PlateCarree()},  # specify plot projection
+)
+
+# first U, save handles so that we can make a nice colorbar later
+handles = plot_labelled_filled_contours(ds.U, ax=ax[0])
+
+# Now V
+plot_labelled_filled_contours(ds.V, ax=ax[1])
+
+cbar = f.colorbar(
+    handles["filled"],  # make colorbar appropriate for this object
+    ax=ax,  # a list of *two* axes, matplotlib will steal space from both these to fit the colorbar
+    orientation="horizontal",  # horizontal colorbars are on the bottom by default
+    aspect=30,  # aspect ratio of colorbar, just because we can.
+)
+cbar.set_ticks(levels)  # set the tick labels on the colorbar
+
+
+# add title for each subplot
+ax[0].set_title("Zonal Wind [m/s]", loc="left")
```

Comment:
Rather than hardcoding the titles can you get them from the 'units' attribute and 'long_name' NetCDF attributes? Something like:

ax.set_title(ds.U.long_name + ' (' + ds.U.units+')',  loc='left')

Reply:
done.